test/MostUsed/MostUsedSpec.hs

```diff
@@ -101,31 +89,68 @@ spec = do
                                         , SingleQuoted "3 x"
                                         , DoubleQuoted "4 y"
                                         , CommandSubstitution "arg"]
-                successes Zsh.items s `shouldBe` [result]
+                successes parser s `shouldBe` [result]
 
             it "parses a line with multiple items separated by a pipe" $ do
                 let s = cmd "c1 one |c2 `two`"
                 let result = [Item "c1" [NotQuoted "one"]
                              , Item "c2" [Backticks "two"]]
-                successes Zsh.items s `shouldBe` result
+                successes parser s `shouldBe` result
 
             it "parses a line with @" $ do
                 let s = cmd "echo @gabebw"
                 let result = [Item "echo" [NotQuoted "@gabebw"]]
 
-                successes Zsh.items s `shouldBe` result
+                successes parser s `shouldBe` result
 
             it "parses a line with $-quoting" $ do
                 let s = cmd "echo $' hello '"
                 let result = [Item "echo" [SingleQuoted " hello "]]
 
-                successes Zsh.items s `shouldBe` result
+                successes parser s `shouldBe` result
 
             it "parses a line with a variable" $ do
                 let s = cmd "echo $var"
                 let result = [Item "echo" [NotQuoted "$var"]]
 
-                successes Zsh.items s `shouldBe` result
+                successes parser s `shouldBe` result
+
+        describe "with multiple items, each of which is on one line" $ do
+            it "parses commands with a variety of arguments" $ do
+                let s1 = cmd "c1 one `two` $(arg)"
+                let s2 = cmd "c2 '3 x' \"4 y\""
+                let s3 = cmd "c3"
+                let s = intercalate "\n" [s1, s2, s3]
+                let i1 = Item "c1" [NotQuoted "one"
+                                   , Backticks "two"
+                                   , CommandSubstitution "arg"]
+                let i2 = Item "c2" [SingleQuoted "3 x"
+                                   , DoubleQuoted "4 y"]
+                successes parser s `shouldBe` [i1, i2, Item "c3" []]
+
+        it "can parse something with a bare escaped newline" $ do
+            let s = cmd "fc -lDt 1 \\n"
+            let item = Item "fc" [NotQuoted "-lDt", NotQuoted "1"]
+
+            successes parser s `shouldBe` [item]
+
+zshSpec :: Parser [Item] -> (String -> String) -> Spec
+zshSpec parser cmd = describe "Zsh-only tests" $ do
+    describe "failures" $ do
+        it "returns a pretty error message" $ do
```

Comment:
It does, and there's [a spec to that effect in commonSpec](https://github.com/gabebw/most-used/pull/13/files#diff-4859476d7e4d672c3ab3a298626c62acR25).

I think this can just be removed tbh